tests/acceptance/features/webUICreateFilesFolders/createFolders.feature

```diff
@@ -92,3 +93,13 @@ Feature: create folders
   Scenario: User creates a folder with .part in the name
     When the user creates a folder with the name "fo.part" using the webUI
     Then folder "fo.part" should be listed on the webUI
+
+  @issue-ocis-2322
+  Scenario: create of new folder into a folder Shares
+    Given user "Brian" has created file "lorem.txt"
```

Comment:
```suggestion
    Given user "Brian" has been created with default attributes and without skeleton files
    And user "Brian" has created file "lorem.txt"
```

We don't need user Brian in all the scenarios of this feature. If Brian is created in the Background section then it wastes CI time on each of the other scenarios.